src/lib/LR1121Driver/LR1121.cpp

```diff
@@ -162,23 +164,82 @@ void LR1121Driver::Config(uint8_t bw, uint8_t sf, uint8_t cr, uint32_t regfreq,
 
     SetMode(LR1121_MODE_STDBY_RC, radioNumber);
 
+    useFSK = setFSKModulation;
+    
     // 8.1.1 SetPacketType
-    uint8_t buf[1] = {LR11XX_RADIO_PKT_TYPE_LORA};
+    uint8_t buf[1] = {useFSK ? LR11XX_RADIO_PKT_TYPE_GFSK : LR11XX_RADIO_PKT_TYPE_LORA};
     hal.WriteCommand(LR11XX_RADIO_SET_PKT_TYPE_OC, buf, sizeof(buf), radioNumber);
 
-    ConfigModParamsLoRa(bw, sf, cr, radioNumber);
+    if (useFSK)
+    {
+        uint32_t Bitrate = bw == LR11XX_RADIO_GFSK_BITRATE_300k ? 300000 : (uint32_t)bw * 1000; // A little hack to get 300 into the uint8_t bw data type
```

Comment:
Might be better just to use a switch statement.
And, what's with these capitalised variable names!

Reply:
I like the suggestion below and will go with x10000.